google/resource_compute_firewall.go

```diff
@@ -302,11 +457,12 @@ func resourceFirewall(
 	}
 
 	// Build the firewall parameter
-	return &compute.Firewall{
+	return &computeBeta.Firewall{
 		Name:         d.Get("name").(string),
 		Description:  d.Get("description").(string),
 		Network:      network.SelfLink,
```

Comment:
Does this self link need to be converted to v1? Also, do we actually need to read the network at both versions or can we just read it at v1 since all we need is the self link?